reton/ave.config.js

```diff
@@ -2,191 +2,121 @@ import Link from "next/link";
 
 const config = {
   Layout: {
-    description: "Ave",
-    title: "Mamador",
-    title2: "Ave resume",
-    twitter: "Ufff we"
+    description: "José Alberto Vera Vigil",
+    title: "Resume",
+    title2: "Ave Resume",
+    twitter: "Ufff we",
   },
   Navbar: {
-    cvLink: "https://raw.githubusercontent.com/eduardosanzb/eduardosanzb/master/cv.pdf",
+    cvLink:
+      "https://raw.githubusercontent.com/eduardosanzb/new-website/master/CV_Jose%20Alberto%20Vera%20Vigil.pdf",
   },
   MainBanner: {
-    bigText: "AVE",
-    firstName: "Alberto",
+    bigText: "ALBERTO",
+    firstName: "José Alberto",
     lastName: "Vera Vigil",
     description: `
-      Hello I am a passionate <span>Software Engineer</span> excited
-      for innovation & technology. With 5+ years of profound and
-      wide-ranging experience. Expert in developing web & mobile
-      solutions. 🇲🇽 🌮
+    I am a passionate Mechanical Engineer excited for innovation & technology.
+    With 7+ years of profound and wide-ranging experience. Expert in product design and development in the automotive industry.
     `,
-    twitter: "eduardosanzb",
-    linkedin: "https://www.linkedin.com/in/eduardosanzb/",
-    github: 'eduardosanzb',
-    clipboard: "npx eduardosanzb",
+    linkedin: "https://www.linkedin.com/in/jose-vigil/",
+    github: "AveVigil",
+    clipboard: "npx AveVigil",
   },
   About: {
-    title: "I'm from Mexico and I love 🌮 & coding<",
+    title: "I'm from Mexico and I love CrossFit, reading, hiking & 🌮",
     description: `
-      I was born in 1993 and grew up in Puebla, 🇲🇽. After graduating as
-      a software engineer I decided to move to CDMX 🇲🇽 and later on
-      Berlin 🇩🇪 the land of techno & cold weather. I freaking 💖 it.
-      <i>
-        (Yeah I know, I'm a city boy and I took the midnight train goin'
-        to kotti)
-      </i>
+    I was born in 1991 and grew up in Puebla, Mexico.
+    After I finished studying as an automotive design engineer, I started working on projects for Volkswagen.
+    Then, I joined Lightyear's mission, where I helped develop the world's first long-range solar electric vehicle that's production-ready.
+    Outside of work you can find me in the GYM or at CorssFit competitions.
+    That's me doing (and almost dying in the process) a 140 kilos clean.
+    I also enjoy reading and backpacking (like any other millennial, haha).
     `,
-    residence: "Berlin",
-    addiction: "☕️ (CAFÉ)",
-    email: "eduardosanzb@gmail.com",
+    residence: "Eindhoven",
+    addiction: "☕️ (espresso)",
+    email: "a.ve.vigil@gmail.com",
   },
   WhatIDo: {
     rows: [
       [
         {
-          title: "Web development",
-          description: "Full stack dev specialized in react, react-native, node and go"
+          title: "Product Development",
+          description:
+            "Automotive components development, from concept to production.",
```

Comment:
Ahora puedes agregar iconos:
Para eso puedes buscar el icono que quieras de [aquí](https://boxicons.com/)
Y si quieres animarlos, puedes usar las opciones de [aquí](https://boxicons.com/usage#animation)

e.g.:

```suggestion
          title: "Product Development",
          description:
            "Automotive components development, from concept to production.",
            icon: "bx bxs-car-mechanic bx-flashing"
```